src/main/java/com/github/princesslana/eriscasper/rest/Route.java

```diff
@@ -31,7 +31,9 @@
     GET("GET", Content.QUERY_STRING),
     PATCH("PATCH", Content.BODY),
     POST("POST", Content.BODY),
-    PUT("PUT", Content.BODY);
+    QUERY_POST("POST", Content.QUERY_STRING),
+    PUT("PUT", Content.BODY),
+    PUT_QUERY("PUT", Content.QUERY_STRING);
```

Comment:
With QUERY_POST or PUT_QUERY (these names are inconsistent ;)), how do I also send the body with that request?

Reply:
I need to rename them yes. Also, it's a different form of request, some of the API stuff need query strings in those places rather than body statements so I just added the ability to use that type of formatting. Not sure if there's a better way to do it.

Reply:
But they don't need just a query string right? Doesn't something like execute slack webhook also need a request body?

Reply:
Nope, just a query string (weird)
https://discordapp.com/developers/docs/resources/webhook#execute-slackcompatible-webhook-querystring-params